File: pyg_portal_web/src/main/webapp/js/service/contentService.js
//服务层
app.service("contentService", function ($http) {

	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../content/findAll.do');
	}
    //分页查询所有
    this.findByPage = function (pageNum,pageSize) {
        return $http.get('../content/findByPage.do?pageNum=' + pageNum + '&pageSize=' + pageSize);
    };
    //根据条件分页查询
    this.findByCondition = function (pageNum, pageSize, conditionObj) {
        return $http.post("../content/findByCondition.do?pageNum=" + pageNum + "&pageSize=" + pageSize, conditionObj);
    };
    //添加
    this.add = function (obj) {
        return $http.post('../content/add.do', obj);
    };
    //根据id查找
    this.findOneById = function (id) {
        return $http.get("../content/findOneById.do?id=" + id);
    };
    //修改
    this.update = function (obj) {
        return $http.post("../content/update.do", obj);
    };
    //批量删除
    this.batchDelete = function (idsArr) {
        return $http.post("../content/batchDelete.do", idsArr);
    };

});

File: pyg_content_interface/src/main/java/com/songxin/pyg/content/service/ContentService.java
package com.songxin.pyg.content.service;

import com.songxin.pyg.pojo.TbContent;
import com.songxin.pyg.vo.PageResultVO;

import java.util.List;
/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface ContentService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<TbContent> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageResultVO findByPage(int pageNum, int pageSize);
	
	
	/**
	 * 增加
	*/
	public void add(TbContent content);
	
	
	/**
	 * 修改
	 */
	public void update(TbContent content);
	

	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public TbContent findOneById(Long id);
	
	
	/**
	 * 批量删除
	 * @param ids
	 */
	public void batchDelete(Long[] ids);

	/**
	 * 根据条件分页查询
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageResultVO findByCondition(TbContent content, int pageNum, int pageSize);


	/**
	 * 根据id查找所有已启动的广告列表 
	 * @param id 
	 * @return java.util.List<com.songxin.pyg.pojo.TbContent>
	 * @author fishsx
	 * @date 2018/12/15 下午8:34
	 */
	public List<TbContent> findAllEnabledByCateId(Long id);
}
